packages/lab/src/drawer/Drawer.css

```diff
@@ -0,0 +1,98 @@
+.saltDrawer {
+  --drawer-boxShadow: var(--saltDrawer-boxShadow, var(--salt-overlayable-shadow-modal));
```

Comment:
doesn't need to be a local token. also, the CSS api is defined here and on the attr (line 24) which is unnecessary - but, I think we aren't adding CSS API tokens to new components atm, so can be deleted and use the characteristic directly :)

Reply:
same for all below